src/features/profile/pages/TrackEmailChange/TrackEmailChange.tsx

```diff
@@ -0,0 +1,133 @@
+import { Step } from 'features/profile/components/Step/Step'
+import { StepCard, StepCardType } from 'features/profile/components/StepCard/StepCard'
+import { StepList } from 'features/profile/components/StepList/StepList'
+import { BackButton } from 'ui/components/headers/BackButton'
+import { BicolorEmailIcon } from 'ui/svg/icons/BicolorEmailIcon'
+import { BicolorNewIcon } from 'ui/svg/icons/BicolorNewIcon'
+import { BicolorPhoneIcon } from 'ui/svg/icons/BicolorPhoneIcon'
+import { getSpacing, Spacer, Typo } from 'ui/theme'
+import { useCustomSafeInsets } from 'ui/theme/useCustomSafeInsets'
+
+type GetTrackEmailStepsProps = {
+  currentEmail: string
+  newEmail: string
+}
+
+export const getTrackEmailSteps = ({ currentEmail }: GetTrackEmailStepsProps) => ({
+  sent: {
+    title: 'Envoie ta demande',
+    icon: <BicolorPhoneIcon />,
+    subtitle: '',
+  },
+  confirm: {
+    title: 'Confirme ta demande',
+    icon: <BicolorEmailIcon />,
+    subtitle: `Depuis l’email envoyé à ${currentEmail}`,
+  },
+  validate: {
+    title: 'Validation de ta nouvelle adresse',
+    icon: <BicolorEmailIcon />,
+    subtitle: 'Sur ta nouvelle adresse blabla',
+  },
+  connect: {
+    title: 'Connexion sur ta nouvelle adresse',
+    icon: <BicolorNewIcon />,
+    subtitle: '',
+  },
+})
+
+const HEADER_HEIGHT = getSpacing(8)
+
+export function TrackEmailChange() {
+  const currentStep = 1 // TODO(dbenfouzari): data should come from API
+
+  const { user } = useAuthContext()
+  const { top } = useCustomSafeInsets()
+  const { goBack } = useGoBack(...homeNavConfig)
+
+  const currentEmail = user?.email ?? ''
+
+  const getStepCardType = useCallback(
+    (stepIndex: number) => {
+      if (stepIndex === currentStep) return StepCardType.ACTIVE
+      if (stepIndex < currentStep) return StepCardType.DONE
+      return StepCardType.DISABLED
+    },
+    [currentStep]
+  )
+
+  return (
+    <StyledScrollViewContainer>
+      <TopSpacer height={HEADER_HEIGHT + top} />
+      <HeaderContainer>
+        <Spacer.TopScreen />
+        <GoBackContainer>
+          <BackButton onGoBack={goBack} />
+        </GoBackContainer>
+      </HeaderContainer>
+      <Spacer.Column numberOfSpaces={6} />
+      <Typo.Title1>Suivi de ton changement d’e-mail</Typo.Title1>
+      <Spacer.Column numberOfSpaces={10} />
+      <StyledContainer>
+        <StyledStepList activeStepIndex={currentStep}>
+          {Object.entries(getTrackEmailSteps({ currentEmail, newEmail: '' })).map(
+            ([stepKey, emailStep], index) => {
+              return (
+                <Step key={stepKey}>
+                  <StyledStepCard
+                    type={getStepCardType(index)}
+                    title={emailStep.title}
+                    subtitle={index === currentStep ? emailStep.subtitle : undefined}
+                    icon={emailStep.icon}
+                  />
+                </Step>
+              )
+            }
+          )}
```

Comment:
Tu connais en avance le nombre d'étapes, ce qu'elles contiennent etc.

Alors plutôt que de faire du code compliqué pour rien, tu peux juste render ici 
```suggestion
          <Step>
            <StepCard
              type={getStepCardType(0)}
              title="Confirme ta demande"
              subtitle={currentStep === 0 ? `Depuis l’email envoyé à ${currentEmail}` : undefined}
              icon={<BicolorEmailIcon />}
            />
          </Step>
          {/* ... les autres étapes */}
            
```